lib/src/component_declaration/component_base/component_base_2.dart

```diff
@@ -227,9 +227,9 @@ abstract class UiComponent2<TProps extends UiProps> extends react.Component2
   //   BEGIN Typed props helpers
   //
 
-  /// A typed props object corresponding to the current untyped props Map ([unwrappedProps]).
+  /// A typed view into the component's current JS props object.
   ///
-  /// Created using [typedPropsFactory] and cached for each Map instance.
+  /// Created using [typedPropsFactoryJs] and updated whenever state changes.
```

Comment:
Aren't there other cases besides `state` updating that would trigger a props update?

Reply:
Good catch; this is a copy-paste error, and should be "whenever props changes"